src/containers/SearchPage/components/results/SearchConcept/ContentView.tsx

```diff
@@ -65,6 +65,10 @@ const ContentView = ({
   const { t } = useTranslation();
   const { data: licenses } = useLicenses();
   const license = licenses && licenses.find((l) => concept.license === l.license);
+  const isGloss = concept.conceptType === 'gloss';
+  const glossInfo = isGloss
+    ? `${t(`languages.${concept.glossData?.originalLanguage}`)}: ${concept.glossData?.gloss}`
+    : '';
```

Comment:
Kunne vi oppdatert denne til:
```
  const conceptInfo = isGloss
    ? `${t(`languages.${concept.glossData?.originalLanguage}`)}: ${concept.glossData?.gloss}`
    : content;
```
Så sparer vi oss for sjekken på isGloss under 😄

Reply:
lurt